packages/core/src/components/forms/controls.tsx

```diff
@@ -129,13 +131,43 @@ const Control: React.SFC<IControlInternalProps> = ({
 // Switch
 //
 
-export interface ISwitchProps extends IControlProps {}
+export interface ISwitchProps extends IControlProps {
+    /**
+     * String to display within the switch control component when the switch is active ("on").
```

Comment:
`active` actually means focused (a la CSS `:active`).
the checkbox state is called `checked` ✅ so let's rename these props.